### ISSUE TITLE
Cannot able to sign in as an architect 

### ISSUE THREAD

**Issue by mohan-m-u** [dmsl/anyplace#231]
Sign-in state: Error
![image](https://user-images.githubusercontent.com/34266758/38452852-f6a4cab6-3a69-11e8-9b51-d7cd74473181.png)

**Comment by Costantinos** [comment]
Hello,

When you click the sign in button do you see the authentication popup window from google?

**Comment by mohan-mu** [comment]
@Costantinos yup  iam getting authentication pop up window and after i sign in i am getting this error

**Comment by ReedSTL** [comment]
We have built a local server using the precomplied code and can log into the server architect on the local machine.  When we try and log on to the local server architect from another PC we get an authentication error below:



400. That’s an error.

Error: redirect_uri_mismatch

The JavaScript origin in the request, http://anyplace.ourdomain.com:9000, does not match the ones authorized for the OAuth client. Visit https://console.developers.google.com/apis/credentials/oauthclient/587500723971-si8s4qqat9v5efgemnebihpi3qe9onlp.apps.googleusercontent.com?project=587500723971 to update the authorized JavaScript origins.

Learn more

Request Details
redirect_uri=storagerelay://http/anyplace.ourdomain.com:9000?id=auth243334
response_type=code permission id_token
scope=openid profile email
openid.realm=
client_id=587500723971-si8s4qqat9v5efgemnebihpi3qe9onlp.apps.googleusercontent.com
ss_domain=http://anyplace.ourdomain.com:9000
access_type=offline
include_granted_scopes=true
prompt=consent
origin=http://anyplace.ourdomain.com:9000
gsiwebsdk=2
That’s all we know.

**Comment by akki12desai** [comment]
can you check if google API key is changed for your google account login?
go to google api console.. replace the client ID i think in  the index.html file

**Comment by exru** [comment]
The same problem!

**Comment by gmkumar2005** [comment]
@exru please provide some more details. Are you getting exactly the same error ?
Do you have any log / screenshot 
Please check the restrictions section in your google console. Trailing slash may cause some errors. remove the trailing slash in all the urls. 

![image](https://user-images.githubusercontent.com/1712742/56711927-2f480280-674a-11e9-9846-d7dfce26d7f6.png)
